javascript/example_code_v3/dynamodb/metadata.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# Amazon Elastic Compute Cloud (ddb) basic usage examples
```

Comment:
(ddb)? does this refer to DynamoDB, or is it the wrong acronym, should be **Amazon EC2**? If latter, then use the short name and no ddb:
**Amazon EC2** basic usage examples